### PR TITLE
Image Segmentation ISIC Melanoma Dataset

### PR DESCRIPTION
**Pull request by JoshuaSYX**
Hi Shakes,

Please review this pull request. My chosen task was Image Segmentation using ISIC Melanoma Dataset using an Improved UNET Model. The number of epochs used was 30, and an average dice coefficient score obtained is 0.94. A description for the attached files are included. 

- **README.md**: Contains useful information about the project
- **driver.py**: Contains the preprocessing functions, training loop and dice coefficient score calculation
- **model.py**: Contains the architecture of the model

Thank you!
Joshua Yu Xuan Soo
S4571796

### CODE REVIEW

**Comment by wooboyeong** [comment]
README
* Please report average DSC for test set in README

General Comments
* Code OK for EASY
* Commit messages OK
* Pull request OK

**Comment by shakes76** [comment]
## TF/Torch Usage
Adequate use and implementation
## Good Practice (Design/Commenting)
Good spacing and comments
Header blocks
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present 
No Data leakage 
Difficulty: Easy -10
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
Please remove model files, these will clog up the repository. Your pull request will not be merged until they are removed. Feedback marks will be awarded when these are removed. -2
Request Description OK

**Comment by JoshuaSYX** [comment]
Hi Shakes,

I think I ran into a slight error, I posted it on Edstem but I will copy it here as well.

I was attempting to fix my project as per the required feedback, but I believed I accidentally pressed on "Fetch Upstream", and my repository on Github was updated but my local repository is not. I only noticed this afterwards and I am now unable to perform any git operations since the 2 repositories are different. 

Is there any advice on how to fix this? Or if possible can I make an entirely new submission because I am unsure on what I should do at this point.

As you can refer above, the 3 "Merge branch __ into __" were not supposed to happen.

Thanks!

**Comment by shakes76** [comment]
Model files are still present though for some reason. Not sure what that error is I'm afraid. Feedback attempted so will give you the marks, but can't merge until solved.

**Comment by JoshuaSYX** [comment]
Hi Shakes,

I managed to fix the problem. As per requested by Boyeong and you, I have added the Dice Score Coefficient in the README file as well as removed the model file that would clog up the repository. Please have a look when you are available. 

Thank you!
